conduct.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Code of conduct
+layout: default
+---
+
+# Code of conduct
+
+---
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
```

Comment:
I don't think including "level of experience" is really appropriate for a technical working group.